app.py

```diff
@@ -50,11 +47,42 @@ def cont():
             db.session.commit()
```

Comment:
db.create_all() is used to create a table and need not be run every time the /register request is called.